lib/bungie_client.dart

```diff
@@ -274,16 +274,16 @@ class BungieClient {
             .map((skull) => skull['displayName']));
   }
 
-  /// Returns the progress on Moments of Triumph as a completion percentage for
+  /// Returns the progress on Age of Triumph as a completion percentage for
```

Comment:
I can certainly do that, but the reason I didn't is the Moments of Triumph event is inactive. You can't progress in it anymore, so it seemed pointless to track it - that information is no longer relevant to anyone.